ansible_runner/config/_base.py

```diff
@@ -87,7 +88,7 @@ def __init__(self,
         if private_data_dir:
             self.private_data_dir = os.path.abspath(private_data_dir)
         else:
-            self.private_data_dir = os.path.abspath(os.path.expanduser('~/.ansible-runner'))
+            self.private_data_dir = os.path.join(gettempdir(), ".ansible-runner")
```

Comment:
okay, so this is `/tmp/.ansible-runner` @matburt suggested out in the issue.

Reply:
err... it would be layered under another tmpdir, so maybe `/tmp/xxxxx/.ansible-runner`

Reply:
I'm a _little_ worried about the possibility of cross-device errors here which ansible tries to do atomic writes.

Reply:
I'm not sure I'm too terribly worried about that relative to the current working directory... `home` could just as easily be off on another device also. The original code, whose provenance is 2.0 adds some new assumptions which, in this case, will require a home directory to be set.